devlab/tests/utils.py

```diff
@@ -123,6 +124,21 @@ class MigrationUtils(object):
 
     def __init__(self, conf):
         self.config = conf
+        self.start_ssh_agent()
+
+    @staticmethod
+    def start_ssh_agent():
+        with hide('everything'), settings(warn_only=True):
+            result = local('ssh-add -l')
+            if result.succeeded:
+                # Add default rsa key to the agent. Key won't be added second
+                # time (in case when key already added)
+                local('ssh-add')
+                return
+            cmd = 'eval `ssh-agent` && echo $SSH_AUTH_SOCK && ssh-add'
+            info_agent = local(cmd, capture=True).split("\n")
+            os.environ["SSH_AGENT_PID"] = info_agent[0].split(" ")[-1]
```

Comment:
Actually will it work without setting environment variables? Maybe these two lines are not needed?..